api/filter_outputs.go

```diff
@@ -41,9 +41,49 @@ func (api *API) updateFilterOutput(w http.ResponseWriter, r *http.Request) {
 			ctx,
 			w,
 			statusCode(err),
-			errors.Wrap(err, "failed to create filter outputs"),
+			errors.Wrap(err, "failed to create filter output"),
 		)
 		return
 	}
 	api.respond.StatusCode(w, http.StatusOK)
 }
+
+//createFilterOutputEvent will add a new event to the list of existing events in the filter outputs
+func (api *API) addFilterOutputEvent(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	fID := chi.URLParam(r, "filter_output_id")
+
+	var req createEventRequest
+
+	if err := api.ParseRequest(r.Body, &req); err != nil {
+		api.respond.Error(
+			ctx,
+			w,
+			http.StatusBadRequest,
+			Error{
+				err: errors.Wrap(err, "invalid request body"),
+				logData: log.Data{
+					"id": fID,
+				},
+			},
+		)
+		return
+	}
+
+	f := model.Event{
```

Comment:
As the request consists of an embedded `model.Event` you can just do `f := req.Event` here.
Also `f` isn't really an appropriate name, `f` was used for filter, `e` would probably be better as it's an event